src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -50,6 +50,27 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   // verify that the user is not reviewing their own change request
   if (reviewerId === foundCR.submitterId) return res.status(401).json({ message: 'Access Denied' });
 
```

Comment:
to respond to your other comment, yeah we should be checking if the change request with crId exists in the first place (kinda a bug that that check doesnt exist already) and you can check that outside the `if foundscopecr` thing because we should check that always

Reply:
I believe that check is already being done around line 45; to clarify the question, I was wondering if I had to ensure that the crId that is in the proposoed solution matches the current crId that is provided/is being reviewed.

Reply:
oof you're right i didn't see that. and yeah that's actually a very good point. you can just include that in the where part of the prisma query (so you have two constraints on the where now instead of one)